runtime/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/model/OpenapiSchemaItem.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021-2024 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.model;
+
+import java.util.List;
+
+import jakarta.json.bind.annotation.JsonbProperty;
+
+public class OpenapiSchemaItem
+{
+    public String type;
+    public String title;
+    public Integer multipleOf;
+    public Integer maximum;
+    public Boolean exclusiveMaximum;
```

Comment:
Support for `exclusiveMaximum` & `exclusiveMinimum` might be tricky as openapi uses Json Schema Draft 4 but the Zilla Json Model uses Draft 7. And in Draft 7, `exclusiveMaximum` & `exclusiveMinimum` parameters are expected to be of type `Integer`.

https://json-schema.org/understanding-json-schema/reference/numeric#range